ui/raidboss/data/06-ew/raid/p3n.txt

```diff
@@ -0,0 +1,110 @@
+## Pandaemonium: Asphedolos, the Third Circle
+
+hideall "--Reset--"
+hideall "--sync--"
+
+0.0 "--Reset--" sync / 21:........:40000010:/ window 100000 jump 0
+
+# -ii 669F 66A7 66A8 66A0 66A1 66AF 66B0 66B3 6D0C 6DDC 6EDD
```

Comment:
66A5 (Joint Pyre) is missed?

Reply:
I was working off a timeline  where I had initially ignored all Sparkfledged adds, so I missed putting it here. Re-added